scripts/lib/CIME/namelist.py

```diff
@@ -2047,12 +2100,13 @@ def _parse_namelist_group(self):
             self._settings[group_name] = {}
         self._eat_whitespace()
         while self._curr() != '/':
-            name, values = self._parse_name_and_values()
+            name, values, addto = self._parse_name_and_values()
             dsettings = []
             if self._groupless:
-                if name in self._settings:
-                    dsettings = self._settings[name]
-                values = merge_literal_lists(dsettings, values)
+                if name in self._settings and not addto:
```

Comment:
In fact, the logic was wrong and I have fixed this. Somehow I missed one of the unit tests failing.